_source/_posts/2022-03-30-java-regex.md

```diff
@@ -0,0 +1,254 @@
+---
+layout: blog_post
+title: "A Quick Guide to Java Regular Expressions"
+author: karl-hughes
+by: contractor
+communities: [java]
+description: ""
+tags: []
+tweets:
+- ""
```

Comment:
```suggestion
- "🛠  - Regular expressions are all around us.
    🔩   - It’s useful to know how to use #regex well. 
     ⚙️  - Build regular expressions in Java, with a little help from our latest tutorial. "
```

Reply:
I don't think it's possible to do multi-line tweets in the front matter.